File: app/src/main/java/com/digicoffer/lauditor/Dashboard/NewRevampViewModels/DashboardNavigationListener.java
package com.digicoffer.lauditor.Dashboard.NewRevampViewModels;

import com.digicoffer.lauditor.Notifications.Models.Navigation;

public interface DashboardNavigationListener {
    void onNavigate(int cardType, Navigation navigation);
    void onNavigate(int cardType, Navigation navigation, String subType);
    void onPaySubscription(String planLabel, String validityText);
}

File: app/src/main/java/com/digicoffer/lauditor/Dashboard/NewRevampViewModels/DashboardNavigationListener.kt
package com.digicoffer.lauditor.Dashboard.NewRevampViewModels

import com.digicoffer.lauditor.Notifications.Models.Navigation

interface DashboardNavigationListener {
    fun onNavigate(cardType: Int, navigation: Navigation?)
    fun onNavigate(cardType: Int, navigation: Navigation?, subType: String?)
    fun onPaySubscription(planLabel: String?, validityText: String?)
}
